iOS/DuckDuckGo/DataBrokerProtectionDebugViewController.swift

```diff
@@ -229,11 +250,27 @@ final class DataBrokerProtectionDebugViewController: UITableViewController {
         case .deviceIdentifier:
             break
 
+        case .deleteAllData:
+            presentDeleteAllDataAlertController()
+
         case .none:
             return
         }
     }
 
+    private func presentDeleteAllDataAlertController() {
+        let alert = UIAlertController(title: "Delete All PIR Data?", message: "This will remove all data and statistics from the PIR database, and reset your Tester ID.", preferredStyle: .alert)
+        alert.addAction(title: "Delete All Data", style: .destructive) { [weak self] in
+            try? self?.manager.database.deleteProfileData()
```

Comment:
This is very tangential and there's nothing actionable here, but I've been wondering about accessing the database through the manager here and just generally thinking what public interface do we want to present from PIR to the rest of the app? I'm keen to avoid the situation on macOS.
I don't have any well developed opinions so far, but my sense is that we probably ought not to be accessing database methods directly

It's much less of a problem now, but we used to have very diffuse decision making on macOS and made some effort to consolidate it